src/test/java/org/biacode/jcronofy/api/impl/CronofyClientImplDeTest.java

```diff
@@ -0,0 +1,2066 @@
+import org.biacode.jcronofy.api.model.request.GetAccessTokenRequest;
+import org.biacode.jcronofy.api.model.request.ListCalendarsRequest;
+import org.biacode.jcronofy.api.model.request.ListNotificationChannelsRequest;
+import org.biacode.jcronofy.api.model.request.ProfileInformationRequest;
+import org.biacode.jcronofy.api.model.request.ReadEventsRequest;
+import org.biacode.jcronofy.api.model.request.RevokeAccessTokenRequest;
+import org.biacode.jcronofy.api.model.request.UpdateAccessTokenRequest;
+import org.biacode.jcronofy.api.model.request.UserInfoRequest;
+import org.biacode.jcronofy.api.model.response.AccountInfoResponse;
+import org.biacode.jcronofy.api.model.response.AvailabilityResponse;
+import org.biacode.jcronofy.api.model.response.BulkDeleteEventsResponse;
+import org.biacode.jcronofy.api.model.response.CloseNotificationChannelResponse;
+import org.biacode.jcronofy.api.model.response.CreateCalendarResponse;
+import org.biacode.jcronofy.api.model.response.CreateNotificationChannelResponse;
+import org.biacode.jcronofy.api.model.response.CreateOrUpdateEventResponse;
+import org.biacode.jcronofy.api.model.response.DeleteEventResponse;
+import org.biacode.jcronofy.api.model.response.FreeBusyResponse;
+import org.biacode.jcronofy.api.model.response.GetAccessTokenResponse;
+import org.biacode.jcronofy.api.model.response.ListCalendarsResponse;
+import org.biacode.jcronofy.api.model.response.ListNotificationChannelsResponse;
+import org.biacode.jcronofy.api.model.response.ProfileInformationResponse;
+import org.biacode.jcronofy.api.model.response.ReadEventsResponse;
+import org.biacode.jcronofy.api.model.response.RevokeAccessTokenResponse;
+import org.biacode.jcronofy.api.model.response.UpdateAccessTokenResponse;
+import org.biacode.jcronofy.api.model.response.UserInfoResponse;
+import org.biacode.jcronofy.api.test.AbstractCronofyUniTest;
+import org.easymock.Mock;
+import org.easymock.TestSubject;
+import org.junit.Before;
+import org.junit.Test;
+
+import javax.ws.rs.BadRequestException;
+import javax.ws.rs.ClientErrorException;
+import javax.ws.rs.ForbiddenException;
+import javax.ws.rs.NotAuthorizedException;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.Invocation;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.GenericType;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.UUID;
+
+import static org.easymock.EasyMock.eq;
+import static org.easymock.EasyMock.expect;
+import static org.easymock.EasyMock.reset;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+/**
+ * User: Arthur Asatryan
```

Comment:
@ChrisPeel-Qudini Did you create this file or was it moved?

Reply:
I duplicated the original test file and changed the endpoints too create this one